packages/map/src/CdcMap.tsx

```diff
@@ -1252,14 +1248,7 @@ const CdcMap = (props: CdcMapProperties) => {
                 vizTitle={general.title}
                 viewport={currentViewport}
                 formatLegendLocation={formatLegendLocation}
-                setFilteredCountryCode={setFilteredCountryCode}
```

Comment:
@adamdoe @Atash3000 @mpallansch I removed this. I'm not sure what it's doing, but I know that it's being called from an onClick handler on a <td> which I don't think is 508 compliant. I'll make a comment there as well

Reply:
@adamdoe and I were talking about this in chat and he said that the feature doesn't even seem to be working in prod. We weren't sure why this control was needed on the datatable either. I would suggest if someone wants to put it back, instead of trying to add a click handler to the outer `<td>` we should instead pass a button or anchor tag into the table that can be clicked. This should indicate to screen readers and regular users that an action is being taken with the click.